modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFieldLocalServiceTest.java

```diff
@@ -411,6 +415,55 @@ public void testAddCustomObjectField() throws Exception {
 					objectDefinition2.getObjectDefinitionId()
 				).build()));
 
+		ObjectFilter objectFilter =
+			_objectFilterLocalService.createObjectFilter(0L);
+
+		objectFilter.setFilterBy(RandomTestUtil.randomString());
+		objectFilter.setFilterType(ObjectFilterConstants.TYPE_EQUALS);
+		objectFilter.setJSON(
+			JSONUtil.put(
+				ObjectFilterConstants.TYPE_EQUALS, RandomTestUtil.randomInt()
+			).toString());
+
+		Assert.assertNotNull(
```

Comment:
@pedro-oliveira446 this test is only testing if we can add a filter with a field that doesn't exist.
Could you create (or increment) a filter/aggregation test to test the case of creating two object definitions respectively: The first one containing the filter/aggregation and the other one containing the field?
Thx.

Reply:
Hey @gabrielwas ,
I’m testing the two validations I removed simultaneously while assigning a random name to the property ObjectFieldSettingConstants.NAME_OBJECT_FIELD_NAME. Since these validations don’t interfere with each other, we can test both at the same time.

Reply:
Exactly. We're just testing here if the validation is removed, but we're not testing if they'll actually work after the secong object is imported. And to make sure we don't break it in the future, would be good having this test.

Reply:
Done!